src/app/components/client/ExposureCard.tsx

```diff
@@ -121,7 +111,7 @@ export const ExposureCard = (props: Props) => {
           >
             <ChevronDown
               className={detailsOpen ? styles.isOpen : ""}
-              alt=""
+              alt={l10n.getString("chevron-down-alt")}
```

Comment:
This now says "Drop down button", which doesn't provide much info about what it does. More descriptive would be something like "Expand" or "Collapse", depending on the current state. In fact, might be useful to add that as a `title` too.

Reply:
I think the same answer as [here](https://github.com/mozilla/blurts-server/pull/3155#discussion_r1247678093) applies: yes, but it gets set by the `alt` prop for icons.

One thing I also noticed while inspecting the chevron in the Accessibility Inspector is this warning:

![image](https://github.com/mozilla/blurts-server/assets/4251/f55be402-ea93-4486-9b3b-0625ad8e68e5)

So I think the `<span>` that has the `onClick` handler should actually be a `<button>`.